.github/workflows/tox.yml

```diff
@@ -3,6 +3,7 @@ name: tox
 on:
   create:  # is used for publishing to PyPI and TestPyPI
     tags:  # any tag regardless of its name, no branches
+      - "**"
```

Comment:
`**` includes slashes, `*` does not. `null` just doesn't set any limitations. Hence it already works like that.

Also, I've learned from my experiments that **you can't really filter out tags vs branches on this level for the `create` event**. So you can as well just drop `tags:`. You'll still have to do manual filtering at the start of the workflow by analyzing the event details: https://github.com/ansible/pylibssh/blob/bd53dd8/.github/workflows/build-test-n-publish.yml#L27-L29

Reply:
Thanks!